src/Vendr.PaymentProviders.Worldpay/WorldpaySettings.cs

```diff
@@ -1,10 +1,58 @@
 using Vendr.Core.Web.PaymentProviders;
 
-namespace Vendr.PaymentProviders
+namespace Vendr.PaymentProviders.Worldpay
 {
     public class WorldpaySettings
     {
-        [PaymentProviderSetting(Name = "Continue URL", Description = "The URL to continue to after this provider has done processing. eg: /continue/")]
+        [PaymentProviderSetting(Name = "Cancel Url", Description = "The Cancel URL")]
+        public string CancelUrl { get; set; }
+
+        [PaymentProviderSetting(Name = "Error Url", Description = "The Error URL")]
+        public string ErrorUrl { get; set; }
+
+        [PaymentProviderSetting(Name = "Continue Url", Description = "The Continue URL")]
         public string ContinueUrl { get; set; }
+
+        [PaymentProviderSetting(Name = "Install ID", Description = "The installation ID")]
+        public string InstallId { get; set; }
+
+        [PaymentProviderSetting(Name = "Mode", Description = "TEST or LIVE")]
+        public string Mode { get; set; }
```

Comment:
Convert this to a boolean. Vendr can handle the conversion https://github.com/vendrhub/vendr-payment-provider-stripe/blob/dev/src/Vendr.PaymentProviders.Stripe/StripeSettingsBase.cs#L80

Reply:
I'd also move this lower down as we tent to put this nearer the bottom. Take a look at some other provider settings.